v1docs/swagger-doc/v2/properties.json

```diff
@@ -1,4 +1,4 @@
 {
   "gaffer.properties.app.title": "Gaffer REST API",
-  "gaffer.properties.app.doc.url": "https://gchq.github.io/gaffer-doc/"
-}
\ No newline at end of file
+  "gaffer.properties.app.doc.url": "https://gchq.github.io/gaffer-doc/v1docs/"
```

Comment:
What does this do?

Reply:
It's used by part of Swagger Doc. The property is [documented here in the Javadoc](https://gchq.github.io/gaffer-doc/latest/javadoc/gaffer/constant-values.html) as `APP_DOCUMENTATION_URL`. It seems this should be set to the URL which is used to host the page. Although I'm not convinced it really makes any difference as the Swagger Doc is working fine with it set to the wrong value.